packages/react/src/components/AccountSettings/ChangePassword/defaultComponents.tsx

```diff
@@ -12,8 +13,28 @@ export const DefaultCurrentPassword: AccountSettingsPasswordField = (props) => {
   return <PasswordField {...props} />;
 };
 
-export const DefaultNewPassword: AccountSettingsPasswordField = (props) => {
-  return <PasswordField {...props} />;
+export const DefaultNewPassword: AccountSettingsPasswordField = ({
+  validationErrors,
+  ...props
+}) => {
+  return (
+    <>
+      <PasswordField {...props} />
+      <ValidationErrors errors={validationErrors} />
```

Comment:
Note that we're using `ValidationErrors` helper component because we need to render `string[]`. `PasswordField` only supports rendering just `string` ([docs](https://ui.docs.amplify.aws/react/components/passwordfield?tab=props)).

Reply:
Can we just pass a `string` with line breaks (`\n`) to prevent the need for `ValidationErrors`?

Reply:
I believe it's better to have them separate for accessibility purposes. If you look at `ValidationErrors`, each error becomes its own `<role="alert"` />` error message rather than one error string.